WordPress/Classes/ViewRelated/Activity/ActivityListViewModel.swift

```diff
@@ -40,17 +40,41 @@ enum ActivityListViewModel {
         case .loading, .error:
             return .Empty
         case .ready(let activities):
-            let rows = activities.map({ activity in
+            var rows = activities.filter({ activity in
+                !activity.isDiscarded
+            }).map({ activity in
                 return ActivityListRow(
                     activity: activity,
                     action: { (row) in
                         presenter.presentRewindFor(activity: activity)
                     }
                 )
             })
-            return ImmuTable(sections: [
-                ImmuTableSection(rows: rows)
-            ])
+
+            guard rows.count > 0 else {
+                return ImmuTable(sections: [])
```

Comment:
This is fine, but just FYI, you can use the shorthand `ImmuTable.Empty` for this

Reply:
👍 good to know!